packages/client/src/Components/Mapping/helpers/MapPlanningUmpireListener.js

```diff
@@ -0,0 +1,139 @@
+import createButton from '../../../Helpers/createDebugButton'
+
+export default class MapPlanningPlayerListener {
```

Comment:
Should this not be `mapPlanningUmpireListener`?

Reply:
Wow, oh yes!